client/client.go

```diff
@@ -16,6 +18,24 @@ import (
 	"golang.org/x/crypto/ssh/agent"
 )
 
+// InstallPublicFiles installs the public part of the cert and key.
+func InstallPublicFiles(prefix string, cert *ssh.Certificate, pub ssh.PublicKey) error {
```

Comment:
I think this is totally up for lawyering but maybe name this `SavePublicFiles()`? I realise `InstallCert` isn't a great name either 😄